Source/ProgressionSystemRuntime/Private/LevelActors/PSStarActor.cpp

```diff
@@ -27,15 +28,27 @@ void APSStarActor::BeginPlay()
 {
 	Super::BeginPlay();
 
+	// Listen to hande when local character is ready 
 	BIND_ON_LOCAL_CHARACTER_READY(this, ThisClass::OnLocalCharacterReady);
+
+	// Listen to handle input for each game state
+	BIND_ON_GAME_STATE_CHANGED(this, ThisClass::OnGameStateChanged);
 }
 
 // Function called every frame on this Actor
 void APSStarActor::Tick(float DeltaTime)
 {
 	Super::Tick(DeltaTime);
 
-	TryPlayHideStarAnimation();
+	// play only if cinematic is started
+	if (StartTimeHideStarsInternal)
+	{
+		TryPlayHideStarAnimation();
+	}
+	if (UMyBlueprintFunctionLibrary::GetMyGameState()->GetCurrentGameState() == ECurrentGameState::Menu)
```

Comment:
your current logic of Menu animation will always override the HideStar animation. It's because cinematic always happening in Menu state. So when Cinematic started, you will play HideStar animation and then Menu Star animation. It's wrong. To fix it, please use `else` statement to avoid player Menu if we play Hide

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/a5231652e33f7f150dc6d860ded531e0a8493cb3